### PR TITLE
[#57] recalc mags

### PR DESCRIPTION
**Pull request by voj**
closes the NZ part of #57 

as discussed, a refactor is probably required to do the opensha part

### CODE REVIEW

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java [hunk 1]
```diff
@@ -68,8 +77,10 @@ public NZSHM22_SubductionInversionTargetMFDs call() throws Exception {
 	}
 
 	public static NZSHM22_InversionFaultSystemRuptSet fromCrustal(FaultSystemRupSet rupSet, U3LogicTreeBranch branch){
-		branch.clearValue(ScalingRelationships.class);
-		branch.setValue(ScalingRelationships.TMG_CRU_2017);
+		if(ScalingRelationships.SHAW_2009_MOD == branch.getValue(ScalingRelationships.class)) {
```
Comment (chrisbc) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java:80]: @voj do we need/want this? e.g. For Crustal SHAW_2009 is a valid option. Let's just do what the user decides is best, no matter how weird ;)

File: .gitignore [hunk 2]
```diff
@@ -33,4 +33,4 @@ bin/**
 # *.pytest
 
 .idea/
-TEST/
+/TEST/
```
Comment (chrisbc) [inline .gitignore:36]: Can you leave the other one (either `TEST/` or `./TEST/` in place please?

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java (hunk 1 above)
Comment (chrisbc) [reply src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java:80]: Oh - I see this is in 'fromSubduction'. Even so,  I don't want to silently override user inputs.

File: .gitignore (hunk 2 above)
Comment (voj) [reply .gitignore:36]: I cannot. My Windows filesystem is case insensitive, and so this line ignores `src/test`

Comment (voj) [reply .gitignore:36]: and `./TEST/` does not ignore the TEST directory :/

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java (hunk 1 above)
Comment (voj) [reply src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java:80]: It's not perfect, I think a proper solution will require us to decide on how we do configurations/logic tree branches. This is already better than before when we would always override any possible user's choice. I think we should wait until we sort out configurations to fix this.

File: .gitignore (hunk 2 above)
Comment (chrisbc) [reply .gitignore:36]: @voj no you're right the `/` here refers to git project root, and NOT the FileSystem root. So it works as it should for me too - thanks

**Pull request merged**